README.md

```diff
@@ -54,15 +54,20 @@ $ sudo teleport start
 In a production environment Teleport must run as root. But to play, just do `chown $USER /var/lib/teleport` 
 and run it under `$USER`, in this case you will not be able to login as someone else though.
 
-If you wish to deploy Teleport inside a Docker container:
+## Docker
 
+### Deploy Teleport
```

Comment:
@benarent I am in the middle of getting answers to get this working, so I will keep this as is for now